apps/meteor/client/lib/utils/getPlanName.ts

```diff
@@ -0,0 +1,28 @@
+export enum PlanName {
+	COMMUNITY = 'Community',
+	STARTER = 'Starter',
+	PRO = 'Pro',
+	PRO_TRIAL = 'Pro_trial',
+	ENTERPRISE = 'Enterprise',
+	ENTERPRISE_TRIAL = 'Enterprise_trial',
+}
+
+export const getPlanName = (isEnterprise: boolean, activeModules: string[], isTrial: boolean): PlanName => {
```

Comment:
is this really the way this is supposed to work? for UI purposes I'm very confident the attribute `tag` from a license should be used.

Reply:
I'm not using this function to display the name of the plan, I'm using the `tag`, I found this necessary to differentiate different scenarios that are tied to the plan... I could have give a different name, return Enum or whatever, but I would have to do this differentiation